example/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "clear": "node ./node_modules/react-native/local-cli/cli.js start --reset-cache",
     "install-pods": "cd ios && pod install && cd ..",
+    "start": "node ./node_modules/react-native/local-cli/cli.js start",
     "test": "jest"
   },
   "dependencies": {
     "react": "16.8.3",
     "react-native": "0.59.5",
-    "react-native-share": "git+https://github.com/react-native-community/react-native-share.git"
+    "react-native-share": "file:../../react-native-share"
```

Comment:
I did this because for me it does makes sense to run against your own clone, which should be up to date or with your local changes